Android/CameraSample/app/src/main/java/com/bentley/sample/camera/ImageCache.kt

```diff
@@ -74,38 +73,34 @@ object ImageCache {
      * @return The image contents if the uri is an image cache uri, null otherwise.
      */
     fun shouldInterceptRequest(uri: Uri): WebResourceResponse? {
-        return uri.takeIf { uri.scheme == urlScheme }?.let {
+        return uri.takeIf { uri.scheme == URL_SCHEME }?.let {
             WebResourceResponse("image/jpeg", "UTF-8", FileInputStream(getFilePath(it)))
         }
     }
 
     /**
      * Gets all the image files cached for a given iModelId.
      * @param params The input parameters, expects an object with an iModelId member.
-     * @return A Json array of cached images uri's, may be empty.
+     * @return A [List] of cached images uri's, may be empty.
      */
-    fun handleGetImages(params: JsonValue?): JsonValue? {
+    fun handleGetImages(params: Map<String, String>): List<String> {
         val files = CameraApplication.instance.applicationContext.getExternalFiles(getDestinationDir(params))
-        return Json.array(*files.map { file ->
+        return files.map { file ->
             getCacheUri(file).toString()
-        }.toTypedArray())
+        }.toTypedArray().toList()
```

Comment:
I believe `map` returns a typed List already.

```suggestion
        }
```

Reply:
Fixed.